lib/Conversion/TTIRToTTIRDecomposition/TTIRToTTIRDecomposition.cpp

```diff
@@ -1099,7 +1099,8 @@ struct ArangeForceLastDimensionPattern
 
     int64_t arangeLength = (end - start) / step;
 
-    ArrayRef<int64_t> ttnnShape = {1, 1, 1, arangeLength};
+    const int64_t requiredShape[] = {1, 1, 1, arangeLength};
+    ArrayRef<int64_t> ttnnShape(&requiredShape[0], 4);
```

Comment:
There is nothing wrong with this, but style-wise I don't think we are using 'naked' C-array anywhere in the project, I would suggest
```cpp
const llvm::SmallVector<int64_t, 4> requiredShape {1, 1, 1, arangeLength};
```
ArrayRef is implicitly constructible from SmallVector, so explicit construction like this with C-array is not needed.

Reply:
Thanks for the suggestion. updated.